zaas-package/src/main/resources/bin/start.sh

```diff
@@ -253,10 +253,10 @@ _BPX_JOBNAME=${ZWE_zowe_job_prefix}${ZAAS_CODE} ${JAVA_BIN_DIR}java \
     -Dapiml.logs.location=${ZWE_zowe_logDirectory} \
     -Dapiml.security.ssl.verifySslCertificatesOfServices=${verifySslCertificatesOfServices:-true} \
     -Dapiml.security.ssl.nonStrictVerifySslCertificatesOfServices=${nonStrictVerifySslCertificatesOfServices:-false} \
-    -Dapiml.security.auth.zosmf.serviceId=${ZWE_configs_apiml_security_auth_zosmf_serviceId:-ibmzosmf} \
-    -Dapiml.security.auth.provider=${ZWE_configs_apiml_security_auth_provider:-zosmf} \
+    -Dapiml.security.auth.zosmf.serviceId=${ZWE_configs_apiml_security_auth_zosmf_serviceId:-${ZWE_configs_apiml_security_auth_zosmf_serviceId:-ibmzosmf}} \
+    -Dapiml.security.auth.provider=${ZWE_configs_apiml_security_auth_provider:-${ZWE_configs_apiml_security_auth_provider:-zosmf}} \
```

Comment:
these should be also `ZWE_components_gateway...`, right?

Reply:
yes

Reply:
I've missed those, thanks!